content/headless/deploy-operator-local.md

```diff
@@ -89,9 +89,9 @@ This is the simplest scenario: you install the Vault operator on a simple cluste
     1. Now you can interact with Vault. For example, add a secret by running `vault kv put secret/demosecret/aws AWS_SECRET_ACCESS_KEY=s3cr3t`
         If you want to access the Vault web interface, open *https://127.0.0.1:8200* in your browser using the root token (to reveal the token, run `echo $VAULT_TOKEN`).
 
-For other configuration examples of the Vault CustomResource, see the YAML files in the [operator/deploy directory of the project](https://github.com/bank-vaults/bank-vaults/tree/master/operator/deploy) (we use these for testing). After you are done experimenting with Bank-Vaults and you want to delete the operator, you can delete the related CRs:
+For other configuration examples of the Vault CustomResource, see the YAML files in the [deploy/examples](https://github.com/bank-vaults/vault-operator/tree/main/deploy/examples) and [test/deploy](https://github.com/bank-vaults/vault-operator/tree/main/test/deploy) directories of the vault-operator repository. After you are done experimenting with Bank-Vaults and you want to delete the operator, you can delete the related CRs:
 
 ```bash
 kubectl delete -f https://raw.githubusercontent.com/banzaicloud/bank-vaults/master/operator/deploy/rbac.yaml
```

Comment:
This should also be changed!

Reply:
Also some lines below there are similar and unchanged lines under the `HA setup with Raft` section.

Reply:
Another thing regarding this page: installing the operator via Helm looks like this now (see also [this PR](https://github.com/bank-vaults/vault-operator/pull/99)):

`helm upgrade --install vault-operator oci://ghcr.io/bank-vaults/helm-charts/vault-operator`

This should be changed two places on this page.

Reply:
Thanks, it seems that we use the helm install command in a number of other places as well, sometimes with different namespaces (see https://github.com/bank-vaults/bank-vaults.dev/pull/118/commits/d3a911b80e59ea1cb878dd852bf1ef497582957b). I don't know if we can simplify these to use a single command/same namespace.